pybossa/view/account.py

```diff
@@ -1182,7 +1182,7 @@ def add_bookmark(user_name, short_name, bookmark_name, bookmark_url, order_by, d
     if bookmark_name is None or len(bookmark_name) > MAX_BOOKMARK_NAME_LEN:
         raise ValueError(f'Bookmark name must be between 1-{MAX_BOOKMARK_NAME_LEN} characters.')
     if bookmark_url is None or len(bookmark_url) > MAX_BOOKMARK_URL_LEN:
-        raise ValueError('Bookmark URL must be between 1-100 characters.')
+        raise ValueError(f'Bookmark URL must be between 1-{MAX_BOOKMARK_URL_LEN} characters.')
```

Comment:
It may also be helpful to include before this line:

```python
msg = f'Bookmark URL must be between 1-{MAX_BOOKMARK_URL_LEN} characters.'
flash(gettext(msg), 'error')
raise ValueError(msg)
```

Reply:
since this is an API call, the `flash` method will not work here.